### ISSUE TITLE
Initial a11y checklist

### ISSUE THREAD

**Issue by d4rekanguok** [researchops/toolbox#32]
- [ ] I’m assuming the sandbox tag is not part of the test, but it’s too low contrast, needs darker text
- [x] The a tag in the nav region that is the current one should have the attribute aria-current="true" so screen reader users know the same info as a sighted user looking at the color background indicator
- [ ] `<h1>` is Welcome and doesn’t seem like the correct description for the page even though it’s the largest text.  Not exactly sure what the right h1 would be, perhaps a hidden one would be best.  I’ll leave it to the designer to “name” the page.  I’m open to it being welcome I suppose considering that it does indeed seem to be a welcome page.
- [x] The “About the project” button at the bottom is a div, best fix is to make it a button html element, other options out there, but I’d stick with a button
- [x] A skip link is needed to skip past the navigation, you can go to the go/tuxedo page and tab into the page to see a live example
- [ ] The buttons that have a selected state need aria-selected="true" added to reflect when they are selected
- [ ] The graphs like “Where research is conducted” are really complex and need a data view somehow added for screen reader users
- [ ] The filters view needs work, I can’t really review it in this short preview
